src/accordion/Accordion.test.js

```diff
@@ -1,21 +1,75 @@
-import test from 'ava';
-import jsdom from 'jsdom';
+import { assert } from 'chai';
 import nunjucks from 'nunjucks';
+import Velocity from 'velocity-animate';
 
 import { Accordion } from './Accordion';
 
-test.beforeEach(t => {
-  global.document = jsdom.jsdom(nunjucks.render(`./Accordion.test.html`));
-});
+let accordion;
 
-// @todo: Write real tests
-test('Can initialize Accordion', t => {
-  Accordion.init('#accordion');
-  t.truthy(Accordion.accordion);
-  t.is(Accordion.transitionDuration, 300);
-});
+const isHidden = (el) => {
+  let style = window.getComputedStyle(el);
+
+  return (style.display === 'none');
+};
+
+describe('Accordion', () => {
+  before(() => {
+    fixture.setBase('src');
+
+  });
+
+  beforeEach(() => {
+    accordion = fixture.load('accordion/Accordion.test.html');
+
+    document.body.insertAdjacentHTML('afterbegin', accordion);
+  });
+
+  afterEach(() => {
```

Comment:
@ArmandoAmador The `beforeEach` is loading the accordion fixture HTML, so it's good to clean it up and reset it to the default in the `afterEach`, just in case there are any DOM changes.